### PR TITLE
Generate the website with JBake

### PR DESCRIPTION
**Pull request by ge0ffrey**
# How it looks

image

# How to use it

```
mvn clean generate-resources
````
GitHub actions takes care of building it and deploying it

# BEFORE MERGING THIS, do the following

- Create a branch called `pages`
- Change the Github Pages settings to use the branch pages.
- Merge this PR to `main`
- Check the Actions tabs at the top, to see if the post-merge job runs and turns green.
- Enjoy the new website!

### CODE REVIEW

File: data/events.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+#
+# Order events by date descending
+
+- eventId: 20221124-brussels
+  meetupUrl: https://www.meetup.com/belgian-java-user-group/events/289451067/
+  location: Brussels
+  date: 2022-11-24
+  talks:
+    - title: "How Java 19 influences the future of your high-scale Java Applications"
+      presenters: Ana-Maria Mihalceanu
```
Comment (ge0ffrey) [inline data/events.yml:20]: Ideally we want to write the presenter's githubId here instead, then add a data/users.yml
like this: https://github.com/kiegroup/optaplanner-website/blob/main/optaplanner-website-root/data/users.yml
and then render it with this macro: https://github.com/kiegroup/optaplanner-website/blob/main/optaplanner-website-root/templates/macros.ftl#L169
to get something like this 
image

**Comment by FDelporte** [comment]
+1 for use a Java site generator :-)

File: .github/workflows/gh-pages-deployment.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+jobs:
+  build-deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-java@v3
+        with:
+          java-version: 17
+          distribution: 'temurin'
```
Comment (FDelporte) [inline .github/workflows/gh-pages-deployment.yml:17]: 'zulu' would look better here :-)

**Review by FDelporte — APPROVED** [review]
Looks great!!

**Comment by TomCools** [comment]
Thanks a lot for the effort @ge0ffrey! :-)

**Pull request merged**